templates/base.html

```diff
@@ -103,7 +112,11 @@
       <h1><a href="{{ SITEURL }}">{{ SITETITLE }}</a></h1>
 
       {% if SITESUBTITLE %}<p>{{ SITESUBTITLE }}</p>{% endif %}
-
+      {% if USE_TIPUE_SEARCH %}
+        <form class="navbar-search" action="/search.html" role="search">
+            <input type="text" name="q" id="tipue_search_input" placeholder="Search..">
```

Comment:
You need to include trans block here, otherwise it does not work with translations.

Reply:
My apologies, I do not understand what you mean by trans block, would you mind please showing me an example of where a trans block is used?

Reply:
Like this `{{ _('Search') }}`.

Reply:
Amended as per latest commit

```suggestion
            {% if USE_TIPUE_SEARCH %} {% extends 'base.html' %} {% block title %} {{ _('Search') }} - {{ SITENAME|striptags }} {% endblock title %} {% block content %}
```